C:\Users\vomin\AppData\Local\Temp\ipykernel_2592\1316080313.py:37: RuntimeWarning: overflow encountered in square
